config/webpack.config.common.js

```diff
@@ -154,5 +154,7 @@ module.exports = {
 
 		// Fancy WebpackBar.
 		new WebpackBar(),
+
+		new DependencyExtractorWebpackPlugin({}),
```

Comment:
```suggestion
		new DependencyExtractorWebpackPlugin( {
			injectPolyfill: false,
			combineAssets: false,
			requestToExternal( request ) {
				if ( request === 'underscore' ) {
					return '_';
				}
			},
		} ),
```

I'm not sure why but the plugin doesn't include underscore so it's missing from the classic editor. This should fix it.

In Distributor I needed to set `injectPolyfill` to true but that might not be needed here.